vertexy/src/private/prefab/PrefabManager.cpp

```diff
@@ -25,7 +30,7 @@ PrefabManager::PrefabManager(ConstraintSolver* inSolver, const shared_ptr<Planar
 	m_grid = inGrid;
 }
 
-void PrefabManager::createPrefab(const vector<vector<Tile>>& inTiles, bool allowRotation, bool allowReflection)
+void PrefabManager::createPrefab(const vector<vector<Tile>>& inTiles, string name /* "" */, bool allowRotation /* false */, bool allowReflection /* false */)

```

Comment:
I've been using wstring (unicode) instead of string, probably worth doing here for consistency sake